okapi-core/src/main/java/org/folio/okapi/service/impl/DockerModuleHandle.java

```diff
@@ -134,7 +134,9 @@ private void logHandler(Buffer b) {
       logSkip = logSkip - b.length();
     }
     if (logBuffer.length() > 0 && logBuffer.charAt(logBuffer.length() - 1) == '\n') {
-      logger.info(id + " " + logBuffer.substring(0, logBuffer.length() - 1));
+      if (logger.isInfoEnabled()) {
+        logger.info("{} {}", id, logBuffer.substring(0, logBuffer.length() - 1));
+      }
```

Comment:
Consider using lambdas to avoid the if-clause: https://logging.apache.org/log4j/2.0/manual/api.html#Java_8_lambda_support_for_lazy_logging

    logger.info("{} {}", () -> id, () -> logBuffer.substring(0, logBuffer.length() - 1));